spec/Packagist/Api/Result/FactorySpec.php

```diff
@@ -79,6 +79,13 @@ function it_creates_packages_with_dependents()
         $this->create($data)->shouldHaveType('Packagist\Api\Result\Package');
     }
 
+    function it_creates_packages_with_null_source()
+    {
+        $data = json_decode(file_get_contents('spec/Packagist/Api/Fixture/get_null_source.json'), true);
```

Comment:
otv

Reply:
Just a copy paste of the other test before this one. I think this class may be re-factored later.